R/load_data.R

```diff
@@ -0,0 +1,106 @@
+# Helper functions not for export
+
+#' Check for a version of data to load
```

Comment:
Consider adding a descriptive title for this function. Currently, this function has the title "Check for a version of data to load". The first line of the function doc, is where the title is extracted from.